src/test/mir-opt/const_prop/mutable_variable_aggregate_partial_read/rustc.main.ConstProp.diff

```diff
@@ -29,17 +29,26 @@
                                            // + ty: i32
                                            // + val: Value(Scalar(0x00000063))
                                            // mir::Constant
-                                           // + span: $DIR/mutable_variable_aggregate_partial_read.rs:6:11: 6:13
+-                                          // + span: $DIR/mutable_variable_aggregate_partial_read.rs:6:11: 6:13
++                                          // + span: $DIR/mutable_variable_aggregate_partial_read.rs:6:5: 6:13
```

Comment:
I should pull 060e437b776e238a492372fbb278fdc27fef1128 out of #71911 and get that merged so we can eliminate this kind of noise from the const prop tests.